ext/dl_test/php_dl_test.h

```diff
@@ -0,0 +1,36 @@
+/*
+  +----------------------------------------------------------------------+
+  | Copyright (c) The PHP Group                                          |
+  +----------------------------------------------------------------------+
+  | This source file is subject to version 3.01 of the PHP license,      |
+  | that is bundled with this package in the file LICENSE, and is        |
+  | available through the world-wide-web at the following url:           |
+  | https://www.php.net/license/3_01.txt                                 |
+  | If you did not receive a copy of the PHP license and are unable to   |
+  | obtain it through the world-wide-web, please send a note to          |
+  | license@php.net so we can mail you a copy immediately.               |
+  +----------------------------------------------------------------------+
+  | Author:                                                              |
+  +----------------------------------------------------------------------+
+*/
+
+#ifndef PHP_DL_TEST_H
+# define PHP_DL_TEST_H
+
+extern zend_module_entry dl_test_module_entry;
+# define phpext_dl_test_ptr &dl_test_module_entry
+
+# define PHP_DL_TEST_VERSION "0.1.0"
```

Comment:
I don't see the point in having an own version for a bundled extension; consider to use `PHP_VERSION` instead of some hard-coded value.